l10n_it_reverse_charge/models/account_invoice.py

```diff
@@ -263,16 +263,17 @@ def reconcile_supplier_invoice(self):
 
     def partially_reconcile_supplier_invoice(self, rc_payment):
         move_line_model = self.env['account.move.line']
+        payment_debit_line = None
         for move_line in rc_payment.line_ids:
-            # testa se nota credito o debito
-            if (self.type == 'in_invoice') and move_line.debit:
+            if move_line.account_id.internal_type == 'payable' and (
```

Comment:
Come mai hai tolto il controllo sul `type`?

Reply:
Perchè, dovendo riconciliare una fattura passiva, prendo sempre e comunque la riga relativa al conto di debito, purchè con importo diverso da zero. Tale riga è quella da recuperare sia in caso di fattura che di nota di credito.

Tra l'altro la precedente implementazione prendeva la riga corretta per caso, nel senso che il recupero dipendeva dall'ordine delle righe in `rc_payment.line_ids` (`payment_debit_line` veniva scritta 2 volte, visto che non c'era il `break`, e la seconda volta era quella giusta, almeno nei casi in cui l'IVA era diversa da zero)